Formula/llvm.rb

```diff
@@ -130,6 +130,7 @@ class Llvm < Formula
   option "with-python", "Build bindings against custom Python"
   option "with-shared-libs", "Build shared instead of static libraries"
   option "without-libffi", "Do not use libffi to call external functions"
+  option "without-experimental", "Do not build libc++experimental library"
```

Comment:
For some reason this formula did not have it at the beginning, so I just keep a possibility to allowing people to keep the old behavior if they want to. But to be honest I do not see any reasons, other than if you already have libc++experimental built somewhere and now this one will give you a conflict